src/components/Table/types.ts

```diff
@@ -18,6 +18,7 @@ export interface columnTypes {
     width?: string;
     minTableWidth?: number;
     sortable?: boolean;
+    footerContent?: string;
```

Comment:
This should be ReactNode so that anything (including Dropdowns) could be passed to the footer